components/analytics/system_analytics/system_analytics.tsx

```diff
@@ -33,7 +34,13 @@ type Props = {
 }
 
 export default class SystemAnalytics extends React.PureComponent<Props> {
-    public componentDidMount() {
+    private pluginStats: Record<string, PluginAnalyticsRow>;
+    public constructor(props: Props) {
+        super(props);
+        this.pluginStats = {};
+    }
```

Comment:
Any reason why this shouldn't be stored in component state or redux?

Also this would be more shorthand:

```suggestion
    private pluginStats: Record<string, PluginAnalyticsRow> = {};
```